Makefile

```diff
@@ -165,10 +166,15 @@ non-kind-deps-update: common-deps-update
 	hack/install-integration-tests-deps.sh non-kind
 
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
+  $(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
+
+operator-sdk: ## Download operator-sdk locally if necessary.
```

Comment:
No. But this is the version our manifests are generated with. I had a different version before, and every time I did `make bundle`, I ended up with a bunch of unnecessarily generated manifests, where the only difference was the operator-sdk version. I thought it will be useful to have the makefile target that allows downloading the correct version